catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/TeamRepository.java

```diff
@@ -165,6 +167,16 @@ private List<EntityReference> getUsers(Team team) throws IOException {
     return EntityUtil.populateEntityReferences(userIds, Entity.USER);
   }
 
+  private Integer getUserCount(UUID teamId) throws IOException {
+    List<EntityRelationshipRecord> userIds = findTo(teamId, TEAM, Relationship.HAS, Entity.USER);
+    int userCount = userIds.size();
+    List<EntityReference> children = getChildren(teamId);
+    for (EntityReference child : children) {
+      userCount += getUserCount(child.getId());
+    }
+    return userCount;
+  }
+
```

Comment:
For a team are we giving the count of all it's children and children's children? Just giving immediate children count is not sufficient?

Reply:
I think this is required show at the landing page of teams how many users are in a parent team. Example Cloud Infra as a department can have multiple teams under it but when a user is looking from org level we can show how many total users under it and how many total teams under it